deploy/001_deploy_hub.ts

```diff
@@ -4,7 +4,17 @@ import { HardhatRuntimeEnvironment } from 'hardhat/types';
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deployer } = await hre.getNamedAccounts();
 
-  await hre.deployments.deploy('Hub', { from: deployer, log: true });
+  if (hre.helpers.isDeployed('Hub')) {
+    return;
+  }
+
+  const Hub = await hre.deployments.deploy('Hub', { from: deployer, log: true });
+
+  hre.helpers.contractDeployments.contracts['Hub'] = {
+    evmAddress: Hub.address,
+    substrateAddress: hre.helpers.convertEvmWallet(Hub.address),
```

Comment:
We don't need substrate address for hub

Reply:
I just added addresses for all contracts